tools/commands/build.py

```diff
@@ -66,37 +90,93 @@ def make(self):
         logger.info("building & installing {}: {}".format(self.name, " ".join(cmd)))
         subprocess.check_call(cmd, env=self.env)
 
+class acbuilder(builder):
+    def __init__(self, name, modules_dir, make_verbose=False):
+        self.make_verbose = make_verbose
+        super(acbuilder, self).__init__(name, modules_dir)
+
+    def clean(self):
+        if os.path.exists(self.build_path):
+            os.system("make -C {} uninstall".format(self.build_path))
+        super(acbuilder, self).clean()
+
+    def prepare(self):
+        os.chdir(self.src_path)
+        cmd = ["./build-tools/autogen.sh"]
```

Comment:
This is actually something specific to safelibc. OK for now, but something to keep in mind if we ever add another autotools package.

Reply:
I have an issue with this: if I run it a second time, I get this error:
```
[build:136] - INFO - ['make'] ['safeclib']
[build:106] - INFO - preparing safeclib: ./build-tools/autogen.sh
WARNING: You have unstaged changes.
Please commit them or use the '-f' option
ASSERT: Command '['./build-tools/autogen.sh']' returned non-zero exit status 1
```
That's because my system has a more recent (i.e. higher serial number) ax_compile_check_sizeof.m4 than the one included in the project.

The solution is: 
```suggestion
        cmd = ["./build-tools/autogen.sh", "-f"]
```

But then I run in a second issue, see below.